webapp/views.py

```diff
@@ -962,3 +962,19 @@ def thank_you():
     return flask.render_template(
         "thank-you.html", referrer=flask.request.args.get("referrer")
     )
+
+
+def get_user_country_by_ip():
+    client_ip = flask.request.headers.get(
+        "X-Real-IP", flask.request.remote_addr
+    )
+    ip_location = ip_reader.get(client_ip)
+    if not ip_location:
+        return flask.jsonify({"client_ip": client_ip, "country_code": None})
+
+    return flask.jsonify(
+        {
+            "client_ip": client_ip,
+            "country_code": ip_location["country"]["iso_code"],
+        }
+    )
```

Comment:
Small style point - it's generally considered less confusing to only have one `return` per function. I wouldn't say it's a hard-and-fast rule, but in this case the double return can be easily avoided. Could I suggest this instead?:

```suggestion
    country_code = None

    if ip_location:
        country_code = ip_location["country"]["iso_code"]

    return flask.jsonify(
        {
            "client_ip": client_ip,
            "country_code": country_code,
        }
    )
```